rabbitmq/include/userver/urabbitmq/broker_interface.hpp

```diff
@@ -72,6 +72,15 @@ class IAdminInterface {
   /// @param deadline execution deadline
   virtual void RemoveQueue(const Queue& queue, engine::Deadline deadline) = 0;
 
+  /// @brief Gets a single message.
+  ///
+  /// @param queue name of the queue
+  /// @param flags queue flags
+  /// @param message consumed message
+  /// @param deadline execution deadline
+  virtual void Get(const Queue& queue, utils::Flags<Queue::Flags> flags,
```

Comment:
> Would it be possible to have this return std::string, instead of using out-parameter?

@itrofimow Yeah that would be possible. Ideally 'Get' should return some kind of custom Message object to store exchange, routing-key, delivery_tag.

also would be nice to be able to get returned value of the call from Awaiter)